packages/block-library/src/query/utils.js

```diff
@@ -62,6 +57,24 @@ export const getEntitiesInfo = ( entities ) => {
 	};
 };
 
+/**
+ * Helper util to return a value from a certain path of the object.
+ * Path is specified as a string of properties, separated by dots,
+ * for example: "parent.child".
+ *
+ * @param {Object} object Input object.
+ * @param {string} path   Path to the object property.
+ * @return {*} Value of the object property at the specified path.
+ */
+const getValueFromObjectPath = ( object, path ) => {
+	const normalizedPath = path.split( '.' );
+	let value = object;
+	normalizedPath.forEach( ( fieldName ) => {
+		value = value?.[ fieldName ];
+	} );
+	return value;
+};
```

Comment:
We're introducing a simple helper here to separate the concerns. It's as simple as possible: supporting only a string path, separated by dots (which was the original behavior).

Reply:
Just wondering if we could use such a helper in other places, too? It seems worth to follow up and/or add a note so we don't forget about it. I know it's just a tiny little helper function but tests may be worthwhile (leaving that to you).

Reply:
I don't think it's worth the effort and packaging, to be honest. Also, one of the benefits of Lodash has been decoupling and IMHO such a simple helper won't bring many benefits for the added cost of additional dependencies and additional coupling a utility package might bring over time.

Reply:
I would love for this to be tested. It seems there aren't a lot of tests for anything in this file though.

Reply:
See #48956